lua/neosolarized.lua

```diff
@@ -127,8 +127,7 @@ function M.setup(opts)
 	Group.new("MoreMsg", colors.blue, colors.none, styles.NONE)
 	Group.new("ModeMsg", colors.blue, colors.none, styles.NONE)
 	Group.new("Question", colors.cyan, colors.none, styles.bold)
-	Group.new("WinSeparator", colors.base00, colors.none, styles.NONE)
-	Group.link("VertSplit", groups.WinSeparator)
+	Group.new("VertSplit", colors.base00, colors.none, styles.NONE)
```

Comment:
Can you remove the changes to this file? Don't think they belong with the rest of the change.

Reply:
Yes its a mistake, I will remove this part as soon as I access my pc as I am outside now.